tools-local/Microsoft.ML.StableApi/Microsoft.ML.StableApi.csproj

```diff
@@ -6,23 +6,23 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.ML" Version="1.4.0" />
-    <PackageReference Include="Microsoft.ML" Version="1.4.0" />
-    <PackageReference Include="Microsoft.ML.DataView" Version="1.4.0" />
-    <PackageReference Include="Microsoft.ML.CpuMath" Version="1.4.0" />
-    <PackageReference Include="Microsoft.ML.FastTree" Version="1.4.0" />
-    <PackageReference Include="Microsoft.ML.LightGbm" Version="1.4.0" />
-    <PackageReference Include="Microsoft.ML.ImageAnalytics" Version="1.4.0" />
-    <PackageReference Include="Microsoft.ML.Mkl.Components" Version="1.4.0" />
-    <PackageReference Include="Microsoft.ML.TimeSeries" Version="1.4.0" />
-    <PackageReference Include="Microsoft.ML.TensorFlow" Version="1.4.0" />
-    <PackageReference Include="Microsoft.ML.OnnxTransformer" Version="1.4.0" />
-    <PackageReference Include="Microsoft.ML.Vision" Version="1.4.0" />
+    <PackageReference Include="Microsoft.Extensions.ML" Version="1.5.2" />
+    <PackageReference Include="Microsoft.ML" Version="1.5.2" />
```

Comment:
We should probably set this to 1.5.0? If we consider the final .x to be a patch release, the stable packages should point to the minor version I think.

Reply:
Sounds good. Fixed.